pkgs/by-name/so/soundalike/package.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildGoModule
+, fetchFromGitea
+, chromaprint
+, makeWrapper
+,
+}:
+
+buildGoModule rec {
+  pname = "soundalike";
+  version = "0.1.2";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "derat";
+    repo = "soundalike";
+    rev = "v0.1.2";
+    hash = "sha256-mpYUVTj3Zll6kNuK5Mdzv1R7k5FZy6XFghhzmAPPVM8=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ chromaprint ];
+  postInstall = ''
+    wrapProgram $out/bin/soundalike \
+      --prefix PATH : ${ lib.makeBinPath [ chromaprint ]}
+  '';
+
+  vendorHash = "sha256-7hRezOBcjB2wsx/SwV519wg3Azh+0kHMcAoc9aYPM3A=";
+  doCheck = true;
+  checkFlags = [
+    "-skip=TestMain_Scan|TestMain_Compare"
+  ];
```

This is a very minor nitpick, but I’d suggest to re‐order the attributes a little.

```suggestion
  vendorHash = "sha256-7hRezOBcjB2wsx/SwV519wg3Azh+0kHMcAoc9aYPM3A=";
  
  nativeBuildInputs = [ makeWrapper ];
  buildInputs = [ chromaprint ];

  doCheck = true;
  checkFlags = [
    "-skip=TestMain_Scan|TestMain_Compare"
  ];

  postInstall = ''
    wrapProgram $out/bin/soundalike \
      --prefix PATH : ${ lib.makeBinPath [ chromaprint ]}
  '';
```

---

Updated here: https://github.com/NixOS/nixpkgs/pull/388332/commits/42e71648fbdeae7a11ae676f79368b53bc7109f0

It makes a lot of sense to put postInstall after check.